project/plugins.sbt

```diff
@@ -22,3 +22,5 @@ libraryDependencies ++= {
 
   defaultPlugins ++ additionalPlugins
 }
+
+addSbtPlugin("com.lightbend.rp" % "sbt-reactive-app" % "0.1.5")
```

Comment:
This doesn't conflict with `sbt-conductr` in the case where it's using the ConductR build target?

Reply:
The dependencies it adds don't seem to be resolvable by the build.

Reply:
Yeah - this was also not intended to be committed. Will be removed as I do final edits on it.